typescript/src/world-company-remuneration/basket-informations.ts

```diff
@@ -0,0 +1,46 @@
+const K = "K";
+
+/**
+ * Created by thomas on 02/12/2019.
+ * This class is a basket
+ */
+
+export class BasketInformations {
```

Comment:
Can we have a better name for this? Do you want it to be a list of just one instance? 
If its a list use something which ends in List